spec/02-integration/04-admin_api/12-plugins-conf_spec.lua

```diff
@@ -31,6 +35,27 @@ describe("Plugins conf property" , function()
       assert.equal(tablex.size(bundled_plugins),
                    tablex.size(json.plugins.available_on_server))
     end)
+    it("expect all plugins are in bundled", function()
+      local res = assert(client:send {
+        method = "GET",
+        path = "/",
+      })
+      local body = assert.res_status(200 , res)
+      local json = assert(cjson.decode(body))
+      local bundled_plugins_list = json.plugins.available_on_server
+      local rocks_installed_plugins, err = utils.execute([[luarocks show kong | grep -oP 'kong\.plugins\.\K([\w-]+)' | uniq]])
+      assert.is_nil(err)
+      local rocks_installed_plugins_list = stringx.split(rocks_installed_plugins, "\n")
+      for _, plugin in ipairs(rocks_installed_plugins_list) do
+        if not NON_BUDLED_PLUGINS[plugin] then
+          assert(bundled_plugins_list[plugin] ~= nil,
+                 "Found installed plugin not in bundled list: " ..
+                 "'" .. plugin .. "'" ..
+                 ", please add it to the bundled list or NON_BUDLED_PLUGINS accordingly."
```

Comment:
```suggestion
                 ", please add it to the bundled list or NON_BUDLED_PLUGINS accordingly."
```
```suggestion
                 ", please add it to the bundled list"
```

Not bundling plugins yet shipping them is an exception and we should keep it that way by not offering that suggestion to the developer.

Reply:
Make sense. Updated